test/services/filters-parser.test.js

```diff
@@ -227,14 +227,15 @@ describe('services > filters-parser', () => {
       const schemaWithFields = {
         fields: [{ field: 'car', reference: 'car.id' }],
       };
-      const filters = '{ "field": "car:brandName", "operator": "starts_with", "value": "Ferrari" }';
+      const filters = { field: 'car:brandName', operator: 'starts_with', value: 'Ferrari' };
       const filtersParser = new FiltersParser(schemaWithFields, timezone, sequelizeOptions);
 
       it('should not be null', async () => {
         expect.assertions(1);
 
         const spy = jest.spyOn(SchemaUtils, 'isSmartField').mockReturnValue(false);
 
+        console.log('Peforming filter');
```

Comment:
est-ce que ce log est utile ?
Si oui 
```suggestion
        console.log('Performing filter');
```

Reply:
Removed